Comment out cairo_debug_reset_static_data for now, until we can fix the asserts that it causes.  b=374680

diff --git a/gfx/thebes/src/gfxPlatform.cpp b/gfx/thebes/src/gfxPlatform.cpp
--- a/gfx/thebes/src/gfxPlatform.cpp
+++ b/gfx/thebes/src/gfxPlatform.cpp
@@ -93,11 +93,14 @@ gfxPlatform::Shutdown()
 
 gfxPlatform::~gfxPlatform()
 {
+#if 0 // Comment this out for now, until we fix bug 374680.
+
     // The cairo folks think we should only clean up in debug builds,
     // but we're generally in the habit of trying to shut down as
     // cleanly as possible even in production code, so call this
     // cairo_debug_* function unconditionally.
     cairo_debug_reset_static_data();
+#endif
 }
 
 PRBool
